contract-examples/hardhat.config.ts

```diff
@@ -52,7 +52,9 @@ export default {
         "0xcdbfd34f687ced8c6968854f8a99ae47712c4f4183b78dcc4a903d1bfe8cbf60",
         "0x86f78c5416151fe3546dece84fda4b4b1e36089f2dbc48496faf3a950f16157c",
         "0x750839e9dbbd2a0910efe40f50b2f3b2f2f59f5580bb4b83bd8c1201cf9a010a"
-      ]
+      ],
+      gas: "auto", 
+      gas_price: "auto",
```

Comment:
I believe defaults are auto. do we need to explicitly provide these?

Reply:
Ah yes no need to explicitly provide these. Thanks!